typescript-api/src/moonbase/interfaces/moon/definitions.ts

```diff
@@ -1,3 +1,6 @@
+// TODO: update default export to make use of all definitions in moonbeam-types-bundle
+// import { moonbeamDefinitions } from "moonbeam-types-bundle";
```

Comment:
Is this needed?

Reply:
Yes, we'll import and use the types bundle in another iteration.